### PR TITLE
OPENEUROPA-1139: Let task-runner manage permissions

### PR DESCRIPTION
**Pull request by dxvargas**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,12 +16,11 @@
         "webmozart/path-util": "^2.3"
     },
     "require-dev": {
-        "drupal/console": "^1.0.2",
```
Comment (ademarco) [inline composer.json]: Any reason why we are removing the console?

Comment (dxvargas) [reply composer.json]: No, I put it back.

File: composer.json [hunk 2]
```diff
@@ -8,21 +8,21 @@
...
         "drupal/drupal-extension": "~4.0",
-        "nikic/php-parser": "~2.0|~3.0|~4.0",
-        "openeuropa/code-review": "~0.3",
+        "nikic/php-parser": "~2.0||~3.0||~4.0",
+        "openeuropa/code-review": "~1.0.0-alpha3",
         "openeuropa/composer-artifacts": "~0.1",
-        "openeuropa/task-runner": "~0.5",
-        "webflo/drupal-core-require-dev": "~8.6"
+        "openeuropa/drupal-core-require-dev": "^8.6",
+        "openeuropa/task-runner": "~1.0@alpha"
```
Comment (ademarco) [inline composer.json]: Task Runner does not have an alpha release, we should change this in `"openeuropa/task-runner": "~1.0@beta"`, to be consistent I'd also change code-review to ` "openeuropa/code-review": "~1.0@alpha",`.

Comment (dxvargas) [reply composer.json]: Done.

**Pull request merged**